virtualization/windowscontainers/quick-start/lift-shift-to-containers.md

```diff
@@ -257,7 +257,7 @@ When validating if the application to be containerized works on Windows containe
 |---|---|---|---|
 |.NET Framework versions 2.X and 3.X|.NET Framework 4.x|.NET Framework 3.5|Windows Server Core|
```

Comment:
IMO this table would benefit from some links to the referenced .NET containers in MAR.